--- system/v2ray@.service.orig	2021-04-06 16:08:45.391888069 +0000
+++ system/v2ray@.service	2021-04-06 16:09:13.632513159 +0000
@@ -8,7 +8,8 @@
 CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
 AmbientCapabilities=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
 NoNewPrivileges=true
-ExecStart=/usr/local/bin/v2ray -config /usr/local/etc/v2ray/%i.json
+Environment=V2RAY_LOCATION_ASSET=/usr/lib64/v2ray
+ExecStart=/usr/bin/v2ray -config /etc/v2ray/%i.json
 Restart=on-failure
 RestartPreventExitStatus=23
 
